docs/content/2.composables/1.useDirectusAuth.md

```diff
@@ -102,6 +102,48 @@ const onSubmit = async () => {
 </script>
 ```
 
+### `inviteUser`
+
+Invite a user to join Directus. **Email and role are required**. Email is sent by directus server to `email` with a link to directus server or `invite_url` containing invite token.
+
+- **Arguments:**
+  - data: [`DirectusInviteCreation`](https://github.com/Intevel/nuxt-directus/blob/main/src/runtime/types/index.d.ts#L239)
+- **Returns:** `Promise<void>`
+
+```vue
+<script setup lang="ts">
+const { inviteUser } = useDirectusAuth();
+const router = useRouter();
+
+const onSubmit = async () => {
+  try {
+    await inviteUser({ email: "", role: "c86c2761-65d3-43c3-897f-6f74ad6a5bd7" });
+  } catch (e) {}
+};
+</script>
+```
+
+### `inviteAccept`
```

Comment:
Maybe rename it to acceptInvite?

Reply:
I was thinking about that, went with invite so that both invite related functions started with "invite" but agree acceptInvite reads nicer.